src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        Color[] colors = Color.values();
+        int index = new Random().nextInt(colors.length);
```

Comment:
make Random a field and use it here. This way we make our code more efficient - no need to recreate random instance per each method call